src/server/enrichment-map/enrichment/enrichment.js

```diff
@@ -74,12 +74,20 @@ const enrichment = (query, userSetting) => {
       const tNameIndex = 11;
       const tDepthIndex = 12;
       const QTListIndex = 13;
+      ret.orderedQuery = formData.ordered_query;
+      ret.userThr = formData.user_thr;
+      ret.minSetSize = formData.min_set_size;
+      ret.maxSetSize = formData.max_set_size;
```

Comment:
The options should not be using snake_case.  Use camelCase

Reply:
These are the parameters from [gprofiler help]<https://biit.cs.ut.ee/gprofiler/help.cgi?help_id=55>. 
Should I do something like, const orderedQuery = 'ordered_query'; ret.orderedQuery = formData[orderedQuery]; ?